leaflet.shpfile.js

```diff
@@ -1,8 +1,11 @@
 L.Shapefile =L.GeoJSON.extend({
-    initialize: function (file, options) {
+    initialize: function (file, options, importUrl) {
         if(typeof cw !== 'undefined'){
+            if (typeof importUrl === 'undefined') {
```

Comment:
importUrl will always be a variable in scope as it's defined in the arguments, cw may or may not be a global so we need to check if anyone has explicitly set it